packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/circle_updates.dart

```diff
@@ -79,10 +79,10 @@ class CircleUpdates {
       }
     }
 
-    addIfNonNull('circlesToAdd', serializeCircleSet(circlesToAdd));
-    addIfNonNull('circlesToChange', serializeCircleSet(circlesToChange));
+    addIfNonNull('circlesToAdd', serializeCircleSet(circlesToAdd!));
+    addIfNonNull('circlesToChange', serializeCircleSet(circlesToChange!));
     addIfNonNull('circleIdsToRemove',
-        circleIdsToRemove.map<dynamic>((CircleId m) => m.value).toList());
```

Comment:
All three of these changes look like they would crash if the relevant variable were null. Are there tests covering this code?